frontend/public/components/row-filter.jsx

```diff
@@ -86,9 +111,14 @@ class CheckBoxes_ extends React.Component {
         toggle={event => this.toggle(event, id)}
       />;
     });
+    const count = this.countNumItems();
 
     return <div className="col-xs-12">
-      <div className="row-filter">{checkboxes}</div>
+      <div className="row-filter">
+        {checkboxes}
+        <a className="co-m-row-filter__selector" onClick={event => this.selectAll(event)}>Select All Filters</a>
```

Comment:
`a` element without an `href` is not accessible. Let's try to make this a `btn-link`.

```suggestion
        <button className="btn btn-link co-m-row-filter__selector" type="button" onClick={this.selectAll}>Select All Filters</button>
```